vendor/vendor-android/ddmlib/src/main/kotlin/com/malinskiy/marathon/android/ddmlib/shell/CliLogcatReceiver.kt

```diff
@@ -24,7 +24,7 @@ class CliLogcatReceiver(
         val logcatFile = createFile()
         val receiver = LogcatParserListener(device, listener)
 
-        process = executeCommandUsingCli(logcatFile, "logcat", "-v", "long")
+        process = executeCommandUsingCli(logcatFile, "logcat", "-v", "long", "-v", "epoch")
```

Comment:
changes the format of logcat timestamps from human-readable to Unix timestamps, so then `LogCatLongEpochMessageParser` can parse the time to `Instant` class